bcipy/helpers/tests/resources/mock_session/session.json

```diff
@@ -0,0 +1,2431 @@
+          0.03518518518518518,
+          0.03518518518518518,
+          0.03518518518518518,
+          0.05,
+          0.03518518518518518
+        ],
+        "eeg_evidence": [
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.059085353538172,
+          1.0,
+          1.0,
+          1.0,
+          1.0122283357109088,
+          0.9462228354620149,
+          1.0141675668872676,
+          0.9406716642242869,
+          0.9468725073189709,
+          0.9493933542119426,
+          0.9447239820713575,
+          1.079737337826649,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          1.0,
+          0.979155982184491,
+          1.0
+        ],
+        "likelihood": [
+          0.03553607387084611,
+          0.034064857111872675,
+          0.03401325085090277,
+          0.03524447355656172,
+          0.03317125789618713,
+          0.038447315380878075,
+          0.033922303359735596,
+          0.03418300186488459,
+          0.03410787483799372,
+          0.03632408855285483,
+          0.03395546326206762,
+          0.036393678389939406,
+          0.03375625797557495,
+          0.033978776912979985,
+          0.0340692381879098,
+          0.03390167650134456,
+          0.03874666741619477,
+          0.0358852714070128,
+          0.0358852714070128,
+          0.0358852714070128,
+          0.0358852714070128,
+          0.0358852714070128,
+          0.0358852714070128,
+          0.0358852714070128,
+          0.0358852714070128,
+          0.0358852714070128,
+          0.04733103000114351,
+          0.0358852714070128
+        ]
+      },
+      "2": {
+        "stimuli": [
+          [
+            "+",
+            "R",
+            "J",
+            "Q",
+            "S",
+            "V",
+            "T",
+            "<",
+            "F",
+            "U",
+            "L"
+          ]
+        ],
+        "eeg_len": 17,
+        "timing_sti": [
+          [
+            0.5,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25,
+            0.25
+          ]
+        ],
+        "triggers": [
+          [
+            "+",
+            0.0
+          ],
+          [
+            "R",
+            0.4668236249126494
+          ],
+          [
+            "J",
+            0.6835600299527869
+          ],
+          [
+            "Q",
+            0.9003684589406475
+          ],
+          [
+            "S",
+            1.1172441079979762
+          ],
+          [
+            "V",
+            1.3339858599938452
+          ],
+          [
+            "T",
+            1.5507312049157917
+          ],
+          [
+            "<",
+            1.767507885931991
+          ],
+          [
+            "F",
+            1.984226447995752
```

Comment:
Since this is a test, we could reduce this to a few inquiries? It could reduce the difficulty in looking over this data in tests